process/factory/interceptorscontainer/shardInterceptorsContainerFactory_test.go

```diff
@@ -14,6 +14,10 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	allShardNumber = 1
```

Comment:
might drop this var as it is not used in the other tests

Reply:
fixed